mail/dovecot2/Portfile

```diff
@@ -105,150 +132,95 @@ variant libstemmer description {Use libstemmer for full-text search} {
     configure.args-append       --with-libstemmer
 }
 
+variant postgresql84 \
```

Comment:
Why keep 8.4, which is EOL, but remove 9.4 and 9.5 which are not?

Reply:
I simply took the greatest minor number in each major and kept it. But I'm -of course- open for suggestions/corrections as to which versions to keep.